Description: Remove --fatal-warnings from build options
Author: Jörg Frings-Fürst <debian@jff.email>
Last-Update: 2019-10-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/meson.build
===================================================================
--- trunk.orig/meson.build
+++ trunk/meson.build
@@ -25,7 +25,6 @@ add_global_arguments(['--target-glib=2.4
                       '--vapidir=@0@'.format(join_paths(meson.current_source_dir(),
                         'vapi')),
                       '--enable-checking',
-                      '--fatal-warnings',
                       '--enable-experimental',
                       '--enable-deprecated'], language : 'vala')
 
